api/client/service.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package nighthawk.client;
+
+import "api/client/options.proto";
+import "api/client/output.proto";
+
+message SendCommandRequest {
+  enum CommandType {
+    kStart = 0;
+    kUpdate = 1;
+    UNSET = 2;
```

Comment:
Nit: the style for proto here would be:
```
enum CommandType {
  UNSET = 0;
  START = 1;
  UPDATE = 2;
}
```
You want the unset/default to be field 0, enum values are always caps.

Reply:
Done